File: Golang/Version3_NumeroALetras/main.go
package main

import (
	"fmt"

	"version3local/numeroaletras"
)

func main() {

	var numero int

	fmt.Print("Ingrese un número: ")
	fmt.Scan(&numero)

	fmt.Println()
	fmt.Println("Número en español:")
	fmt.Println(numeroaletras.Convertir(numero))

}

File: Rust/Version3_NumeroALetras/src/main.rs
use std::io;
use spanish_numbers::{NumberToSpanish, ScaleType};

fn main() {

    println!("Ingrese un número:");

    let mut entrada = String::new();

    io::stdin().read_line(&mut entrada).unwrap();

    let numero: u128 = entrada.trim().parse().unwrap();

    // Escala larga (España y la usada normalmente en español)
    let conversor = NumberToSpanish::new(ScaleType::Long);

    let texto = conversor.number_to_spanish(numero, " ");

    println!();

    println!("Número en español:");

    println!("{}", texto);

}